config/values/bursaries.yml

```
@@ -16,6 +16,7 @@ bursaries:
     languagesother: £26,000
     maths: £29,000
     music: £10,000
+    nonuk: £31,000
```

```suggestion
    nonuk: £29,000
```
Only scholarships go up to £31k - the highest bursary is £29k?